content/en/serverless/glossary/_index.md

```diff
@@ -0,0 +1,91 @@
+| Step Functions          | Step Functions is a service provided by AWS which offers a way of orchestrating common workflows composed of multiple Lambda functions or cloud service events, without needing to write plumbing code to handle the workflow state and retry logic, etc.                     |
+| Deployment package | Lambda function code can be deployed using a deployment package: either a ZIP archive that contains the function code and dependencies, or a container image that is compatible with the [Open Container Initiative (OCI)][1] specification. |
+| Edge location         | An AWS data center used to perform service-specific operations.                      |
+| Event | A JSON document that contains data for a Lambda function to process. |
+| Lambda function | A serverless function in Lambda. Each function had code to process events and can be invoked to run. |
+| Lambda layer | A ZIP archive that contains additional code—for example, libraries, a custom runtime, configuration files, or other dependencies. You can use Lambda layers to use libraries in your serverless functions without having to include these libraries in your deployment package. |
+| Managed policy | An IAM policy that can be attached to multiple users, groups, and roles. These can be created and managed by AWS or by a customer.|
+| Resource | An S3 bucket, EC2 instance, IAM user, or other entity that can be used in AWS. |
+| Resource property| When including a resource in an AWS CloudFormation stack, each resource can have one or more associated properties. |
+| Serverless Application Model (SAM)          | SAM is an Infrastructure as Code framework developed by AWS specifically focused on serverless applications.                                |
+
+
+
+### Datadog serverless for AWS Lambda concepts
+
+| Concept                         | Description                                                                                                                                                                                                          |
+|---------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Enhanced Lambda metrics | Enhanced Lambda metrics give you a view above and beyond the default Lambda metrics enabled with the AWS Lambda integration. These metrics are distinguished by being in the `aws.lambda.enhanced.*` namespace, and are Datadog’s best practice for setting real-time monitors on your serverless application health.|
+| Lambda library       | The Datadog Lambda Library is responsible for generating real-time enhanced Lambda metrics, submitting custom metrics, and enabling APM and Distributed Tracing for Node.js, Python, and Ruby                          |
```

Comment:
The Datadog Lambda Library collects and submits data (enhanced lambda metrics, traces, etc.) from your Lambda function runtime either to logs (to be picked up by the Forwarder) or to the Datadog Extension.

^ not the exact language, just the idea. Better connect the dot between the Library and Forwarder/Extension.